node/tls.ts

```diff
@@ -1 +1,47 @@
-throw new Error('"tls" is not yet implemented.');
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+// Copyright Joyent and Node contributors. All rights reserved. MIT license.
+
+import { notImplemented } from "./_utils.ts";
+
+export class CryptoStream {}
+export class SecurePair {}
+export class Server {}
+export class TLSSocket {}
+export function checkServerIdentity() {
+  notImplemented();
+}
+export function connect() {
+  notImplemented();
+}
+export function createSecureContext() {
+  notImplemented();
+}
+export function createSecurePair() {
+  notImplemented();
+}
+export function createServer() {
+  notImplemented();
+}
+export function getCiphers() {
+  notImplemented();
+}
+export const rootCertificates = undefined;
```

Comment:
Side discussion: open question how to support this. Node uses a hard-coded list of root certificates, based off Mozilla's NSS certdata.txt.

We use `rustls_native_certs::load_native_certs()` but that produces different results on different platforms. For reasons of compatibility, we may want to use node's list instead.

Reply:
maybe we should add an option to `Deno.ConnectTlsOptions` for completely swap the root certificates? https://doc.deno.land/deno/stable/~/Deno.ConnectTlsOptions

Reply:
Yes, I agree. That'd be useful for more than just node compat too.

Reply:
@bnoordhuis `rustls_native_certs::load_native_certs()` no we don't. We use the same mozilla list by default. You can switch to your native cert list by setting `DENO_TLS_CA_STORE=system` or `DENO_TLS_CA_STORE=system,mozilla` to use both.

Reply:
Oh, that's right - I believe I even reviewed the PR that added that. 🤦

Then perhaps we want to bypass `DENO_TLS_CA_STORE=system` and stick to the built-in list.